components/Button/__tests__/__snapshots__/Button.test.jsx.snap

```diff
@@ -15,7 +15,7 @@ exports[`Button renders with a custom color theme correctly 1`] = `
   line-height: 2rem;
   padding-left: 1.4rem;
   padding-right: 1.4rem;
-  color: #f4f141;
```

Comment:
This change in snapshot is because it is no longer possible to control font colors. It will be decided automatically based on the background color

Reply:
woah nicee ✨